### PR TITLE
#1 create serverless notarization api

### LINKED ISSUE #1: Create serverless notarization API

### LINKED ISSUE #2: Create Action to auto publish the notarization API on push to master
Whenever there is a push to this repo, on the master branch, the functions app should be published to the corresponding Azure Function.

### PR DESCRIPTION
**Pull request by skolumban-ex**

### CODE REVIEW

File: ObjectHashServerr.API/Function1.cs [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+using Microsoft.OpenApi.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using ObjectHashServer.BLL.Models;
+using ObjectHashServer.BLL.Models.Api.Request;
+using ObjectHashServer.BLL.Models.Api.Response;
+using ObjectHashServer.BLL.Services.Implementations;
+using ObjectHashServer.Controllers;
+
+namespace ObjectHashServerr.API
```
Comment (weigandf) [inline ObjectHashServerr.API/Function1.cs]: The name has a typo: ObjectHashServer with one r at the end - I guess ;)

File: ObjectHashServerr.API/Function1.cs [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+{
+    public class Function1
```
Comment (weigandf) [inline ObjectHashServerr.API/Function1.cs]: Guess Function1 will be replaced with a meaningfull name

**Review by weigandf — CHANGES_REQUESTED** [review]
Small typo ObjectHashServerr - rest looks good. Thanks

File: ObjectHashServerr.API/Function1.cs (hunk 2 above)
Comment (skolumban-ex) [reply ObjectHashServerr.API/Function1.cs]: @tbordi-ex  rename the class and file to ObjectHasherFunctions

**Comment by skolumban-ex** [comment]
@tbordi-ex, once you fix the comments from @weigandf , you can merge into the master and delete this branch. Then merge the master into the branch of #5 , so the diff will be clean there.

Reassing that review to Florian, when the diff is only with respect to the most recent master.

**Comment by weigandf** [comment]
@skolumban-ex @tbordi-ex any update?

**Comment by skolumban-ex** [comment]
@weigandf , @tbordi-ex will work on the project again tomorrow, it will be actualized then.

**Pull request merged**